application/controllers/tags_controller.py

```diff
@@ -13,6 +13,32 @@ class TagsController(ApplicationController):
 
     method_decorators = [check_es_connection]
 
+    @use_args(tags_validator.multi_tagging_args())
+    def multi_tagging(self, params):
+        """ Add tags to multiple artifacts """
+
+        new_multi_tags = list(params["tags"])
+        existing_multi_tags = []
+        artifacts = []
+
+        for object_id in set(params["ids"]):
+            try:
+                artifact = Artifact.find(object_id)
+                artifacts.append(artifact)
+
+                if not existing_multi_tags:
+                    existing_multi_tags = artifact.tags
+                else:
+                    existing_multi_tags = list(set(existing_multi_tags) & set(artifact.tags))
```

Comment:
Why do you always convert to list?
I think it would be better to initialize existiting_multi_tags as an empty set.